opus/import/obs_cassini_common_pds4.py

```
@@ -0,0 +1,459 @@
+        return rev_no
+
+    def field_obs_mission_cassini_rev_no(self):
+        return self._create_mult_keep_case(self._rev_no())
+
+    def field_obs_mission_cassini_rev_no_int(self):
+        rev_no = self._rev_no()
+        if rev_no is None:
+            return None
+        try:
+            rev_no_cvt = opus_support.parse_cassini_orbit(rev_no)
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)